core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -192,12 +221,10 @@ public void loadRegularThenFaultTolerant() throws Exception {
     }
   }
 
-  /**
-   * Ensures that Hadoop loads the Alluxio file system when configured.
-   */
   @Test
   public void hadoopShouldLoadFileSystemWhenConfigured() throws Exception {
     org.apache.hadoop.conf.Configuration conf = getConf();
+    ConfigurationTestUtils.resetConfiguration();
```

Comment:
As @aaudiber  suggested, if updating `ConfigurationRule` to support `null` is easy to fix, I will prefer fixing `ConfigurationRule`

Reply:
ConfigurationRule supports null, and pass in Map instead of immutable map.